Controller/Payment/Order.php

```diff
@@ -25,7 +50,12 @@ public function __construct(
         \Magento\Checkout\Model\Session $checkoutSession,
         \Razorpay\Magento\Model\CheckoutFactory $checkoutFactory,
         \Razorpay\Magento\Model\Config $config,
-        \Magento\Catalog\Model\Session $catalogSession
+        \Magento\Catalog\Model\Session $catalogSession,
+        \Magento\Quote\Model\QuoteRepository $quoteRepository,
+        \Magento\Sales\Api\Data\OrderInterface $order,
+        \Magento\Quote\Model\QuoteManagement $quoteManagement,
+        \Magento\Store\Model\StoreManagerInterface $storeManagement,
+        \Magento\Customer\Api\CustomerRepositoryInterface $customerRepository
```

Comment:
@ChetanGN , can you please just try use `Magento\Quote\Api\CartManagementInterface` to place the order this much code not required all the details already present in quote in active session. No need to fetch anything or form anything.